config/config.exs

```diff
@@ -114,6 +114,10 @@ config :esbuild,
          js/app.js
          js/storybook.js
          js/editor/Editor.tsx
+         js/react/components/DataclipViewer.tsx
```

Comment:
Ouch, so this is needed to allow the `jsx()` macro thing to work in the production build?

We don't need to do it now, but I think we should do a deep dive on this and find a better solution. Because this is just an invitation for nasty bugs to creep in

Reply:
Would love to see a demo, but I reckon this is the lessor of two evils because _if_ you don't up date this list, it won't work in dev or prod.

While we shouldn't _have_ to have everything be it's own entrypoint, we do need to be super mindful of what module imports what - we've had more than once the typescript compiler make it's way into modules that change frequently; resulting in massive assets for users.